src/components/NationalText/utils.ts

```diff
@@ -0,0 +1,62 @@
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last } from 'lodash';
+import moment from 'moment';
+import { formatPercent, formatDecimal } from 'common/utils';
+
+// Fix all uses of !
+
+const getPercentChange = (series: any[]): number => {
+  const lastTwoWeeks = series.slice(-14);
+  const difference = last(lastTwoWeeks)! - lastTwoWeeks[0]!;
+  const percentChange = difference / lastTwoWeeks[0]!;
+  return percentChange;
+};
```

Comment:
I would recommend implementing a `percentChange` function that just takes 2 numbers so we can use it in other places

```ts
function percentChange(a: number, b: number): number | null {
  const validParams = isFinite(a) && isFinite(b) && b !== 0;
  return validParams ? a / b : null;
}
```